UnitsNet/CustomCode/UnitClasses/Force.extra.cs

```diff
@@ -70,5 +70,10 @@ public static Force FromMassByAcceleration(Mass mass, double metersPerSecondSqua
         {
             return new Force(mass.Kilograms*metersPerSecondSquared);
         }
+
+        public static Force FromMassByAcceleration(Mass mass, Acceleration acceleration)
```

Comment:
Not sure you noticed, but the buildserver failed on this overload when compiling the WinRT solution. If you are not familiar with WinRT Components (I wasn't until I was asked to support it), it has very strict requirements on public members in order to support WinJS and C++ UWP apps.

Anyhow. we can't have overloads in WinRT, so I suggest we wrap the `double` overload in a `#if !WINDOWS_UWP  ... #endif` to exclude it from WinRT. It would be a breaking change for that target, but I know for a fact that the number of WinRT users is very low and the chance they are using this particular method is even lower. That one guy, if he exists, can deal with the breaking change. YOLO!